app/webpacker/controllers/modal_link_controller.js

```diff
@@ -30,4 +33,16 @@ export default class extends Controller {
   getIdentifier() {
     return "modal";
   }
+
+  #setPathToFormAction(confirmButton) {
+    const isSubmitButton = confirmButton.type === 'submit';
```

Comment:
Just wondering, is there a case where this would be anything other than `submit`?

Reply:
Yes, David, it would be as button_to will always render a button with submit type.
Never mind this 😅  as I'll be reverting this controller's changes in the future commits soon. Actually had to incorporate the slide-out animation and these changes are not helpful in that case. Sorry for inconvenience 😅